.env.example

```
@@ -6,4 +6,8 @@ REDIS_HOST=
 REDIS_PORT=
 REDIS_DATABASE=
 JWT_SECRET=
-JWT_ACCESS_EXPIRATION_MS=
\ No newline at end of file
+JWT_ACCESS_EXPIRATION_MS=
+
+AWS_ACCESS_KEY=
+AWS_SECRET_KEY=
+AWS_S3_BUCKET
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)